modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -61,17 +62,22 @@ object types {
     remainingWeight: Weight
   )
 
+  case class OneTimeReward(epoch: EpochProgress, address: Address, txnAmount: TransactionAmount)
+
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
-    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch
+    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
+    oneTimeRewards: List[OneTimeReward] = List.empty
```

Comment:
I think it shouldn't stay `empty` 🤔 there should be a one time reward for stardust, right?

Reply:
@marcinwadon If we leave it empty then how will we populate it?

Reply:
This was put back to _empty_ and we'll do the one-time transfer at a later date.